pkg/controller/sidecarset/sidecarset_controller.go

```diff
@@ -131,7 +130,29 @@ func (r *ReconcileSidecarSet) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err
 	}
 
-	// update sidecarset status
 	err = r.updateSidecarSetStatus(sidecarSet, status)
```

Comment:
Should we move the status update in the end of the method and add stats about how many pods have been upgraded in the new version?

Reply:
in fact, it makes no difference to move it to the end of this function. because we  calculate status from informer cache, which won't get updated until next sync loop.